Gandalan.IDAS.Logging/Logging/Logger.cs

```diff
@@ -24,15 +24,15 @@ public class Logger : ILogger
 
         public Logger()
         {
-            LogLevels = new Dictionary<LogContext, Logging.LogLevel>();
+            LogLevels = new Dictionary<LogContext, LogLevel>();
             SetLogDateiPfad();
         }
 
         public void SetLogDateiPfad(string pfad = null)
         {
-            string datum = DateTime.Today.ToString("dd-MM-yyy");
-            string app = Path.GetFileNameWithoutExtension(Assembly.GetEntryAssembly()?.CodeBase ?? "WebApi");
-            string user = Environment.UserName;
+            var datum = DateTime.Today.ToString("yyyy-MM-dd");
+            var app = Path.GetFileNameWithoutExtension(Assembly.GetEntryAssembly()?.CodeBase ?? "WebApi");
```

Comment:
CodeBase is deprecated afaik, better use Location

Reply:
Fixed `SYSLIB0012`